BlogTemplate/Pages/Drafts.cshtml

```diff
@@ -18,10 +18,10 @@
                     <h2><a asp-page="/edit" asp-route-id="@post.Id" asp-route-slug="@post.Slug">@post.Title</a></h2>
                     @if (post.PublishTime != default(DateTimeOffset))
                     {
-                        <div class="text-muted">Last Published @post.PublishTime.ToString("MMM dd, yyyy")</div>
+                        <div class="h5 text-muted">Last Published @post.PublishTime.ToString("MMM dd, yyyy")</div>

```

Comment:
Just wondering, why did you use `class="h5"` instead of using an `h5` tag?

Reply:
Originally I had used an h5 tag, but I wanted to make the HTML semantics proper, since PublishTime is metadata rather than a heading and better suited to a div tag than an h5 tag. To get the same text styling, I'm using h5 Bootstrap class.